### PR TITLE
1337: Site level derived data in Total Cases indicators for PSSS

### PR DESCRIPTION
**Pull request by rohan-bes**
### Issue https://github.com/beyondessential/tupaia-backlog/issues/1337:

Added support for site level derived data in Total Cases indicators for PSSS

### CODE REVIEW

File: packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    code: 'sumSiteWeeklyCases',
+    config: {
+      formula: `PSSS_${syndromeCode}_Cases`,
+      aggregation: [
+        'FINAL_EACH_WEEK',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'facility',
+            aggregationEntityType: 'country',
```
Comment (rohan-bes) [inline packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js:40]: One might argue that we ought to use `requested` here, but given that we currently only care about aggregating to the country level I'd rather be explicit and we can change it later if needs be

File: packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    [`PSSS_${syndromeCode}_Daily_Cases`]: ['SUM_EACH_WEEK', 'FINAL_EACH_WEEK'],
+    sumSiteWeeklyCases: 'RAW',
+    sumSiteDailyCases: 'RAW',
+  },
+  defaultValues: {
+    [`PSSS_${syndromeCode}_Cases`]: 'undefined',
+    [`PSSS_${syndromeCode}_Daily_Cases`]: 'undefined',
+    sumSiteWeeklyCases: 'undefined',
+    sumSiteDailyCases: 'undefined',
+  },
```
Comment (kael89) [inline packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js:81]: We should add support for a single string (expanded for all formula variables) for `defaultValues` at some point... We already support this for `aggregation`

**Review by kael89 — APPROVED** [review]
Nice work! So happy it worked smoothly out of the box!! 😌

File: packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js (hunk 2 above)
Comment (rohan-bes) [reply packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js:81]: Or even a default default value?

Comment (kael89) [reply packages/database/src/migrations/20210415110921-UpdatePSSSWeeklyIndicatorsSupportSiteData-modifies-data.js:81]: Very meta, haha! We probably don't want to use `default` values by default - we have to use it here to overcome the "all defined or not calculate" logic in indicators

**Pull request merged**